pkg/es/config/config.go

```diff
@@ -69,9 +69,12 @@ type Configuration struct {
 	IndexPrefix                    string         `mapstructure:"index_prefix"`
 	IndexDateLayoutSpans           string         `mapstructure:"-"`
 	IndexDateLayoutServices        string         `mapstructure:"-"`
+	IndexDateLayoutSampling        string         `mapstructure:"-"`
```

Comment:
do we not need to define templates for the new indices you're introducing? `plugin/storage/es/mappings/`

Reply:
Yes, we do need it. I am working on it.